package.json

```diff
@@ -10,7 +10,6 @@
     "author": "chrisrhymes",
     "license": "MIT",
     "dependencies": {
-        "bulma": "^0.9.3",
-        "bulma-block-list": "^0.7"
```

Comment:
Hi @misaelnieto I've noticed bulma-block-list has been removed. I have now released a v1 that should work with bulma v1. Are you able to add this new version back in?

Reply:
Hi @chrisrhymes, Nice to know bulma-block-list is now bulmav1 compatible.

When adding back bulma-block-list I got this error:

```
bundle exec jekyll build
Configuration file: /home/nnieto/Code/Hkl/bulma-clean-theme/_config.yml
            Source: /home/nnieto/Code/Hkl/bulma-clean-theme
       Destination: /home/nnieto/Code/Hkl/bulma-clean-theme/_site
 Incremental build: disabled. Enable with --incremental
      Generating... 
       Jekyll Feed: Generating feed for posts
Error: Can't find stylesheet to import.
  ╷
5 │ @import "bulma/sass/utilities/initial-variables.scss";
  │         ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  ╵
  /home/nnieto/Code/Hkl/bulma-clean-theme/node_modules/bulma-block-list/src/block-list.scss 5:9  @import
  _main.scss 12:9                                                                                @import
  /home/nnieto/Code/Hkl/bulma-clean-theme/assets/css/app.scss 2:9                                root stylesheet 
  Conversion error: Jekyll::Converters::Scss encountered an error while converting 'assets/css/app.scss':
                    Can't find stylesheet to import.
                    ------------------------------------------------
      Jekyll 4.3.3   Please append `--trace` to the `build` command 
                     for any additional information or backtrace. 
                    ------------------------------------------------
/home/nnieto/Code/Hkl/bulma-clean-theme/vendor/bundle/ruby/3.2.0/gems/jekyll-sass-converter-3.0.0/lib/jekyll/converters/scss.rb:175:in `rescue in convert': Can't find stylesheet to import. (Jekyll::Converters::Scss::SyntaxError)

        raise SyntaxError, e.message
              ^^^^^^^^^^^^^^^^^^^^^^
        from /home/nnieto/Code/Hkl/bulma-clean-theme/vendor/bundle/ruby/3.2.0/gems/jekyll-sass-converter-3.0.0/lib/jekyll/converters/scss.rb:159:in `convert'
        from /home/nnieto/Code/Hkl/bulma-clean-theme/vendor/bundle/ruby/3.2.0/gems/jekyll-4.3.3/lib/jekyll/renderer.rb:105:in `block in convert'
        from /home/nnieto/Code/Hkl/bulma-clean-theme/vendor/bundle/ruby/3.2.0/gems/jekyll-4.3.3/lib/jekyll/renderer.rb:104:in `each'
        from /home/nnieto/Code/Hkl/bulma-clean-theme/vendor/bundle/ruby/3.2.0/gems/jekyll-4.3.3/lib/jekyll/renderer.rb:104:in `reduce'
        from /home/nnieto/Code/Hkl/bulma-clean-theme/vendor/bundle/ruby/3.2.0/gems/jekyll-4.3.3/lib/jekyll/renderer.rb:104:in `convert'
        from /home/nnieto/Code/Hkl/bulma-clean-theme/vendor/bundle/ruby/3.2.0/gems/jekyll-4.3.3/lib/jekyll/renderer.rb:84:in `render_document'
        from /home/nnieto/Code/Hkl/bulma-clean-theme/vendor/bundle/ruby/3.2.0/gems/jekyll-4.3.3/lib/jekyll/renderer.rb:63:in `run'
        from /home/nnieto/Code/Hkl/bulma-clean-theme/vendor/bundle/ruby/3.2.0/gems/jekyll-4.3.3/lib/jekyll/site.rb:572:in `render_regenerated'
        from /home/nnieto/Code/Hkl/bulma-clean-theme/vendor/bundle/ruby/3.2.0/gems/jekyll-4.3.3/lib/jekyll/site.rb:564:in `block in render_pages'
        from /home/nnieto/Code/Hkl/bulma-clean-theme/vendor/bundle/ruby/3.2.0/gems/jekyll-4.3.3/lib/jekyll/site.rb:563:in `each'
        from /home/nnieto/Code/Hkl/bulma-clean-theme/vendor/bundle/ruby/3.2.0/gems/jekyll-4.3.3/lib/jekyll/site.rb:563:in `render_pages'
        from /home/nnieto/Code/Hkl/bulma-clean-theme/vendor/bundle/ruby/3.2.0/gems/jekyll-4.3.3/lib/jekyll/site.rb:211:in `render'
        from /home/nnieto/Code/Hkl/bulma-clean-theme/vendor/bundle/ruby/3.2.0/gems/jekyll-4.3.3/lib/jekyll/site.rb:80:in `process'
        from /home/nnieto/Code/Hkl/bulma-clean-theme/vendor/bundle/ruby/3.2.0/gems/jekyll-4.3.3/lib/jekyll/command.rb:28:in `process_site'
        from /home/nnieto/Code/Hkl/bulma-clean-theme/vendor/bundle/ruby/3.2.0/gems/jekyll-4.3.3/lib/jekyll/commands/build.rb:65:in `build'
        from /home/nnieto/Code/Hkl/bulma-clean-theme/vendor/bundle/ruby/3.2.0/gems/jekyll-4.3.3/lib/jekyll/commands/build.rb:36:in `process'
        from /home/nnieto/Code/Hkl/bulma-clean-theme/vendor/bundle/ruby/3.2.0/gems/jekyll-4.3.3/lib/jekyll/command.rb:91:in `block in process_with_graceful_fail'
        from /home/nnieto/Code/Hkl/bulma-clean-theme/vendor/bundle/ruby/3.2.0/gems/jekyll-4.3.3/lib/jekyll/command.rb:91:in `each'
        from /home/nnieto/Code/Hkl/bulma-clean-theme/vendor/bundle/ruby/3.2.0/gems/jekyll-4.3.3/lib/jekyll/command.rb:91:in `process_with_graceful_fail'
        from /home/nnieto/Code/Hkl/bulma-clean-theme/vendor/bundle/ruby/3.2.0/gems/jekyll-4.3.3/lib/jekyll/commands/build.rb:18:in `block (2 levels) in init_with_program'
        from /home/nnieto/Code/Hkl/bulma-clean-theme/vendor/bundle/ruby/3.2.0/gems/mercenary-0.4.0/lib/mercenary/command.rb:221:in `block in execute'
        from /home/nnieto/Code/Hkl/bulma-clean-theme/vendor/bundle/ruby/3.2.0/gems/mercenary-0.4.0/lib/mercenary/command.rb:221:in `each'
        from /home/nnieto/Code/Hkl/bulma-clean-theme/vendor/bundle/ruby/3.2.0/gems/mercenary-0.4.0/lib/mercenary/command.rb:221:in `execute'
        from /home/nnieto/Code/Hkl/bulma-clean-theme/vendor/bundle/ruby/3.2.0/gems/mercenary-0.4.0/lib/mercenary/program.rb:44:in `go'
        from /home/nnieto/Code/Hkl/bulma-clean-theme/vendor/bundle/ruby/3.2.0/gems/mercenary-0.4.0/lib/mercenary.rb:21:in `program'
        from /home/nnieto/Code/Hkl/bulma-clean-theme/vendor/bundle/ruby/3.2.0/gems/jekyll-4.3.3/exe/jekyll:15:in `<top (required)>'
        from /home/nnieto/Code/Hkl/bulma-clean-theme/vendor/bundle/ruby/3.2.0/bin/jekyll:25:in `load'
        from /home/nnieto/Code/Hkl/bulma-clean-theme/vendor/bundle/ruby/3.2.0/bin/jekyll:25:in `<top (required)>'
        from /home/nnieto/Code/Hkl/bulma-clean-theme/vendor/bundle/ruby/3.2.0/gems/bundler-2.3.15/lib/bundler/cli/exec.rb:58:in `load'
        from /home/nnieto/Code/Hkl/bulma-clean-theme/vendor/bundle/ruby/3.2.0/gems/bundler-2.3.15/lib/bundler/cli/exec.rb:58:in `kernel_load'
        from /home/nnieto/Code/Hkl/bulma-clean-theme/vendor/bundle/ruby/3.2.0/gems/bundler-2.3.15/lib/bundler/cli/exec.rb:23:in `run'
        from /home/nnieto/Code/Hkl/bulma-clean-theme/vendor/bundle/ruby/3.2.0/gems/bundler-2.3.15/lib/bundler/cli.rb:483:in `exec'
        from /home/nnieto/Code/Hkl/bulma-clean-theme/vendor/bundle/ruby/3.2.0/gems/bundler-2.3.15/lib/bundler/vendor/thor/lib/thor/command.rb:27:in `run'
        from /home/nnieto/Code/Hkl/bulma-clean-theme/vendor/bundle/ruby/3.2.0/gems/bundler-2.3.15/lib/bundler/vendor/thor/lib/thor/invocation.rb:127:in `invoke_command'
        from /home/nnieto/Code/Hkl/bulma-clean-theme/vendor/bundle/ruby/3.2.0/gems/bundler-2.3.15/lib/bundler/vendor/thor/lib/thor.rb:392:in `dispatch'
        from /home/nnieto/Code/Hkl/bulma-clean-theme/vendor/bundle/ruby/3.2.0/gems/bundler-2.3.15/lib/bundler/cli.rb:31:in `dispatch'
        from /home/nnieto/Code/Hkl/bulma-clean-theme/vendor/bundle/ruby/3.2.0/gems/bundler-2.3.15/lib/bundler/vendor/thor/lib/thor/base.rb:485:in `start'
        from /home/nnieto/Code/Hkl/bulma-clean-theme/vendor/bundle/ruby/3.2.0/gems/bundler-2.3.15/lib/bundler/cli.rb:25:in `start'
        from /home/nnieto/Code/Hkl/bulma-clean-theme/vendor/bundle/ruby/3.2.0/gems/bundler-2.3.15/exe/bundle:48:in `block in <top (required)>'
        from /home/nnieto/Code/Hkl/bulma-clean-theme/vendor/bundle/ruby/3.2.0/gems/bundler-2.3.15/lib/bundler/friendly_errors.rb:117:in `with_friendly_errors'
        from /home/nnieto/Code/Hkl/bulma-clean-theme/vendor/bundle/ruby/3.2.0/gems/bundler-2.3.15/exe/bundle:36:in `<top (required)>'
        from /usr/bin/bundle:25:in `load'
        from /usr/bin/bundle:25:in `<main>'
/home/nnieto/Code/Hkl/bulma-clean-theme/vendor/bundle/ruby/3.2.0/gems/sass-embedded-1.74.1-x86_64-linux-gnu/lib/sass/compiler/host.rb:86:in `compile_request': Can't find stylesheet to import. (Sass::CompileError)
        from /home/nnieto/Code/Hkl/bulma-clean-theme/vendor/bundle/ruby/3.2.0/gems/sass-embedded-1.74.1-x86_64-linux-gnu/lib/sass/compiler.rb:171:in `compile_string'
        from /home/nnieto/Code/Hkl/bulma-clean-theme/vendor/bundle/ruby/3.2.0/gems/sass-embedded-1.74.1-x86_64-linux-gnu/lib/sass/embedded.rb:37:in `compile_string'
        from /home/nnieto/Code/Hkl/bulma-clean-theme/vendor/bundle/ruby/3.2.0/gems/jekyll-sass-converter-3.0.0/lib/jekyll/converters/scss.rb:160:in `convert'
        from /home/nnieto/Code/Hkl/bulma-clean-theme/vendor/bundle/ruby/3.2.0/gems/jekyll-4.3.3/lib/jekyll/renderer.rb:105:in `block in convert'
        from /home/nnieto/Code/Hkl/bulma-clean-theme/vendor/bundle/ruby/3.2.0/gems/jekyll-4.3.3/lib/jekyll/renderer.rb:104:in `each'
        from /home/nnieto/Code/Hkl/bulma-clean-theme/vendor/bundle/ruby/3.2.0/gems/jekyll-4.3.3/lib/jekyll/renderer.rb:104:in `reduce'
        from /home/nnieto/Code/Hkl/bulma-clean-theme/vendor/bundle/ruby/3.2.0/gems/jekyll-4.3.3/lib/jekyll/renderer.rb:104:in `convert'
        from /home/nnieto/Code/Hkl/bulma-clean-theme/vendor/bundle/ruby/3.2.0/gems/jekyll-4.3.3/lib/jekyll/renderer.rb:84:in `render_document'
        from /home/nnieto/Code/Hkl/bulma-clean-theme/vendor/bundle/ruby/3.2.0/gems/jekyll-4.3.3/lib/jekyll/renderer.rb:63:in `run'
        from /home/nnieto/Code/Hkl/bulma-clean-theme/vendor/bundle/ruby/3.2.0/gems/jekyll-4.3.3/lib/jekyll/site.rb:572:in `render_regenerated'
        from /home/nnieto/Code/Hkl/bulma-clean-theme/vendor/bundle/ruby/3.2.0/gems/jekyll-4.3.3/lib/jekyll/site.rb:564:in `block in render_pages'
        from /home/nnieto/Code/Hkl/bulma-clean-theme/vendor/bundle/ruby/3.2.0/gems/jekyll-4.3.3/lib/jekyll/site.rb:563:in `each'
        from /home/nnieto/Code/Hkl/bulma-clean-theme/vendor/bundle/ruby/3.2.0/gems/jekyll-4.3.3/lib/jekyll/site.rb:563:in `render_pages'
        from /home/nnieto/Code/Hkl/bulma-clean-theme/vendor/bundle/ruby/3.2.0/gems/jekyll-4.3.3/lib/jekyll/site.rb:211:in `render'
        from /home/nnieto/Code/Hkl/bulma-clean-theme/vendor/bundle/ruby/3.2.0/gems/jekyll-4.3.3/lib/jekyll/site.rb:80:in `process'
        from /home/nnieto/Code/Hkl/bulma-clean-theme/vendor/bundle/ruby/3.2.0/gems/jekyll-4.3.3/lib/jekyll/command.rb:28:in `process_site'
        from /home/nnieto/Code/Hkl/bulma-clean-theme/vendor/bundle/ruby/3.2.0/gems/jekyll-4.3.3/lib/jekyll/commands/build.rb:65:in `build'
        from /home/nnieto/Code/Hkl/bulma-clean-theme/vendor/bundle/ruby/3.2.0/gems/jekyll-4.3.3/lib/jekyll/commands/build.rb:36:in `process'
        from /home/nnieto/Code/Hkl/bulma-clean-theme/vendor/bundle/ruby/3.2.0/gems/jekyll-4.3.3/lib/jekyll/command.rb:91:in `block in process_with_graceful_fail'
        from /home/nnieto/Code/Hkl/bulma-clean-theme/vendor/bundle/ruby/3.2.0/gems/jekyll-4.3.3/lib/jekyll/command.rb:91:in `each'
        from /home/nnieto/Code/Hkl/bulma-clean-theme/vendor/bundle/ruby/3.2.0/gems/jekyll-4.3.3/lib/jekyll/command.rb:91:in `process_with_graceful_fail'
        from /home/nnieto/Code/Hkl/bulma-clean-theme/vendor/bundle/ruby/3.2.0/gems/jekyll-4.3.3/lib/jekyll/commands/build.rb:18:in `block (2 levels) in init_with_program'
        from /home/nnieto/Code/Hkl/bulma-clean-theme/vendor/bundle/ruby/3.2.0/gems/mercenary-0.4.0/lib/mercenary/command.rb:221:in `block in execute'
        from /home/nnieto/Code/Hkl/bulma-clean-theme/vendor/bundle/ruby/3.2.0/gems/mercenary-0.4.0/lib/mercenary/command.rb:221:in `each'
        from /home/nnieto/Code/Hkl/bulma-clean-theme/vendor/bundle/ruby/3.2.0/gems/mercenary-0.4.0/lib/mercenary/command.rb:221:in `execute'
        from /home/nnieto/Code/Hkl/bulma-clean-theme/vendor/bundle/ruby/3.2.0/gems/mercenary-0.4.0/lib/mercenary/program.rb:44:in `go'
        from /home/nnieto/Code/Hkl/bulma-clean-theme/vendor/bundle/ruby/3.2.0/gems/mercenary-0.4.0/lib/mercenary.rb:21:in `program'
        from /home/nnieto/Code/Hkl/bulma-clean-theme/vendor/bundle/ruby/3.2.0/gems/jekyll-4.3.3/exe/jekyll:15:in `<top (required)>'
        from /home/nnieto/Code/Hkl/bulma-clean-theme/vendor/bundle/ruby/3.2.0/bin/jekyll:25:in `load'
        from /home/nnieto/Code/Hkl/bulma-clean-theme/vendor/bundle/ruby/3.2.0/bin/jekyll:25:in `<top (required)>'
        from /home/nnieto/Code/Hkl/bulma-clean-theme/vendor/bundle/ruby/3.2.0/gems/bundler-2.3.15/lib/bundler/cli/exec.rb:58:in `load'
        from /home/nnieto/Code/Hkl/bulma-clean-theme/vendor/bundle/ruby/3.2.0/gems/bundler-2.3.15/lib/bundler/cli/exec.rb:58:in `kernel_load'
        from /home/nnieto/Code/Hkl/bulma-clean-theme/vendor/bundle/ruby/3.2.0/gems/bundler-2.3.15/lib/bundler/cli/exec.rb:23:in `run'
        from /home/nnieto/Code/Hkl/bulma-clean-theme/vendor/bundle/ruby/3.2.0/gems/bundler-2.3.15/lib/bundler/cli.rb:483:in `exec'
        from /home/nnieto/Code/Hkl/bulma-clean-theme/vendor/bundle/ruby/3.2.0/gems/bundler-2.3.15/lib/bundler/vendor/thor/lib/thor/command.rb:27:in `run'
        from /home/nnieto/Code/Hkl/bulma-clean-theme/vendor/bundle/ruby/3.2.0/gems/bundler-2.3.15/lib/bundler/vendor/thor/lib/thor/invocation.rb:127:in `invoke_command'
        from /home/nnieto/Code/Hkl/bulma-clean-theme/vendor/bundle/ruby/3.2.0/gems/bundler-2.3.15/lib/bundler/vendor/thor/lib/thor.rb:392:in `dispatch'
        from /home/nnieto/Code/Hkl/bulma-clean-theme/vendor/bundle/ruby/3.2.0/gems/bundler-2.3.15/lib/bundler/cli.rb:31:in `dispatch'
        from /home/nnieto/Code/Hkl/bulma-clean-theme/vendor/bundle/ruby/3.2.0/gems/bundler-2.3.15/lib/bundler/vendor/thor/lib/thor/base.rb:485:in `start'
        from /home/nnieto/Code/Hkl/bulma-clean-theme/vendor/bundle/ruby/3.2.0/gems/bundler-2.3.15/lib/bundler/cli.rb:25:in `start'
        from /home/nnieto/Code/Hkl/bulma-clean-theme/vendor/bundle/ruby/3.2.0/gems/bundler-2.3.15/exe/bundle:48:in `block in <top (required)>'
        from /home/nnieto/Code/Hkl/bulma-clean-theme/vendor/bundle/ruby/3.2.0/gems/bundler-2.3.15/lib/bundler/friendly_errors.rb:117:in `with_friendly_errors'
        from /home/nnieto/Code/Hkl/bulma-clean-theme/vendor/bundle/ruby/3.2.0/gems/bundler-2.3.15/exe/bundle:36:in `<top (required)>'
        from /usr/bin/bundle:25:in `load'
        from /usr/bin/bundle:25:in `<main>'
```

Any idea what I'm doing wrong?

Reply:
I have been looking into this. It seems that it can be fixed by adding the below to the sass settings in the `_config.yml` so sass knows where to get the files from. 

```yaml
sass:
  load_paths:
    - node_modules
````

I have been testing this out and published a [beta release of the theme gem](https://rubygems.org/gems/bulma-clean-theme/versions/1.0.0.beta.1), but I am unable to get this working when it is used as a theme in another Jekyll site. The load_paths only load seem to load files from within the folder and not from the themes folder.